cocos/platform/apple/CCFileUtils-apple.mm

```diff
@@ -60,7 +60,8 @@ static void addItemToArray(id item, ValueVector& array)
         const char* numType = [num objCType];
         if(num == (void*)kCFBooleanFalse || num == (void*)kCFBooleanTrue)
         {
-            array.push_back(Value([num boolValue]));
+            bool v = [num boolValue];
+            array.push_back(Value(v));
```

Comment:
What's the error?
Is there any differences between the modified codes and previous.